FuelSDK-CSharp/ETClient.cs

```diff
@@ -91,8 +94,12 @@ public ETClient(NameValueCollection parameters = null, RefreshState refreshState
                 {
                     configSection.RestEndPoint = parameters["restEndPoint"];
                 }
+                if (parameters.AllKeys.Contains("oAuth2AuthenticationMode"))
+                {
+                    configSection.RestEndPoint = parameters["oAuth2AuthenticationMode"];
```

Comment:
Should be configSection.OAuth2AuthenticationMode = parameters["oAuth2AuthenticationMode"];

Reply:
Done.